### PR TITLE
[global CSS work] Encapsulate DateTimePicker with global CSS

### PR DESCRIPTION
**Pull request by W3stside**
Moves `global.ts` styles specific to component into accepted `GlobalStyles` wrap/encapsulation method

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1542--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1542--dexreact.review.gnosisdev.com/storybook/)**: Component stories

File: src/components/TimePicker.tsx [hunk 1]
```diff
@@ -36,18 +37,21 @@ const DateTimePickerBase: React.FC<DateTimePickerBase> = ({
...
-      // inputFormat="yyyy/MM/dd HH:mm"
-      ampm={false}
-      minDateTime={minDateTime}
-      renderInput={(props): JSX.Element => (
-        <TextField {...props} label="Set custom date" name={inputName} error={!!error} />
-      )}
-    />
+    <>
+      <GlobalStyles />
+      <MobileDateTimePicker
```
Comment (Velenir) [inline src/components/TimePicker.tsx:42]: Btw, you can pass `className` to `MobileDateTimePicker`, which means can wrap it in `styled(MobileDateTimePicker)`.
That class gets applied to `.MuiPickersBasePicker-container`.
Will it be enough to cover all the global styles? Probably not

File: src/components/TimePicker.tsx [hunk 2]
```diff
@@ -36,18 +37,21 @@ const DateTimePickerBase: React.FC<DateTimePickerBase> = ({
...
+        {...restProps}
+        dateAdapter={memoizedDateAdapter}
+        disablePast
+        minutesStep={BATCH_TIME / 60}
+        // inputFormat="yyyy/MM/dd HH:mm"
+        ampm={false}
+        minDateTime={minDateTime}
+        renderInput={(props): JSX.Element => (
+          <TextField {...props} label="Set custom date" name={inputName} error={!!error} />
```
Comment (Velenir) [inline src/components/TimePicker.tsx:51]: Can also pass `className` here

**Review by Velenir — APPROVED** [review]

File: src/components/TimePicker.tsx (hunk 1 above)
Comment (W3stside) [reply src/components/TimePicker.tsx:42]: > Will it be enough to cover all the global styles? Probably not

:joy: :point_up: my fear

**Review by anxolin — APPROVED** [review]

**Pull request merged**